pkg/cli/internal/frameworks/typescript/templates/tests/echo.test.ts.tmpl

```diff
@@ -0,0 +1,30 @@
+import { echo } from '../src/tools/echo.js';
+
+describe('echo tool', () => {
+  it('should echo back the message', async () => {
+    const result = await echo.handler({ message: 'Hello, World!' });
+    
+    expect(result.content).toHaveLength(1);
+    expect(result.content[0].type).toBe('text');
+    expect(result.content[0].text).toBe('Echo: Hello, World!');
+  });
+
+  it('should convert message to uppercase when uppercase is true', async () => {
+    const result = await echo.handler({ 
+      message: 'Hello, World!', 
+      uppercase: true 
+    });
+    
+    expect(result.content).toHaveLength(1);
+    expect(result.content[0].type).toBe('text');
+    expect(result.content[0].text).toBe('Echo: HELLO, WORLD!');
+  });
+
+  it('should use default uppercase value when not provided', async () => {
+    const result = await echo.handler({ message: 'Hello, World!' });
+    
+    expect(result.content).toHaveLength(1);
+    expect(result.content[0].type).toBe('text');
+    expect(result.content[0].text).toBe('Echo: Hello, World!');
```

Comment:
in `echo.ts.tmpl` the echo handler returns an object with the message and timestamp while we expect result here to contain a content array with the attributes type and text. Could we update these tests to align with the implementation?